runtime/doc/treesitter.txt

```diff
@@ -429,10 +429,10 @@ add_predicate({name}, {handler}, {force})                    *add_predicate()*
                                be (match, pattern, bufnr, predicate)
 
 get_node_text({node}, {source})                              *get_node_text()*
-                Gets the text corresponding to a given node
+                Gets the text corresponding to a given node or a list of nodes
 
                 Parameters: ~
-                    {node}    the node
+                    {node}    the node or list of nodes
```

Comment:
What is a return value/type when a list of nodes is passed? List of strings or a single string?

Reply:
Tbh, thinking about it, do we even need this change. What's the reason for allowing this function to accept a node list as opposed to just running a map function on the node list?